From 0c498f963b413a9f5eb569a4da829b96a9ca233e Mon Sep 17 00:00:00 2001
From: Xu Qiang <xuqiang36@huawei.com>
Date: Fri, 3 Dec 2021 18:15:44 +0800
Subject: [PATCH] Export sysboml for bbox to use.
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 0c498f963b413a9f5eb569a4da829b96a9ca233e


ascend inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4D63I
CVE: NA

-------------------------------------------------

Export console_flush_on_panic for bbox to use.

Signed-off-by: Xu Qiang <xuqiang36@huawei.com>
Signed-off-by: Fang Lijun <fanglijun3@huawei.com>
Reviewed-by: Ding Tianhong <dingtianhong@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 kernel/printk/printk.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/printk/printk.c b/kernel/printk/printk.c
index b9c63109acab..16cb3837b3a5 100644
--- a/kernel/printk/printk.c
+++ b/kernel/printk/printk.c
@@ -2647,6 +2647,7 @@ void console_flush_on_panic(enum con_flush_mode mode)
 	}
 	console_unlock();
 }
+EXPORT_SYMBOL(console_flush_on_panic);
 
 /*
  * Return the console tty driver structure and its associated index
-- 
2.26.2

